zephyr/port/host_flash/cache_utils.c

```diff
@@ -132,7 +152,7 @@ void IRAM_ATTR spi_flash_disable_interrupts_caches_and_other_cpu(void)
 
 void IRAM_ATTR spi_flash_enable_interrupts_caches_and_other_cpu(void)
```

Comment:
I think we are missing `IRAM_ATTR bool spi_flash_cache_enabled(void)` custom call. Can you check?

Reply:
The implementation for s_cache_enabled was added so that spi_flash_cache_enabled(void) is complete for C6